scipy/optimize/tests/test_milp.py

```diff
@@ -273,6 +273,47 @@ def test_infeasible_prob_16609():
     np.testing.assert_equal(res.status, 2)
 
 
+_msg_time = "Time limit reached. (HiGHS Status 13:"
+_msg_iter = "Iteration limit reached. (HiGHS Status 14:"
+
+
+@pytest.mark.skipif(np.intp(0).itemsize < 8,
+                    reason="Unhandled 32-bit GCC FP bug")
+@pytest.mark.slow
```

Comment:
```suggestion
@pytest.mark.slow
@pytest.mark.timeout(360) 
```
i added a greater timeout to allow azure to find at least one feasible solution

Reply:
The test checks if limiting via iteration also works properly but this take too much time. Maybe we should have a better test to check that (one that is faster) but this should fix azure tests

Reply:
If we can think up another problem that has the property of consistently showing iteration limit reached/timeout reached then we can swap it in (or I suppose we could split it into two tests if we needed different problems).  Happy to raise the timeout to see if that fixes it -- this is a "slow" test after all